ic-os/guestos/envs/dev/BUILD.bazel

```diff
@@ -59,3 +61,16 @@ artifact_bundle(
     prefix = "guest-os/update-img-dev",
     visibility = ["//visibility:public"],
 )
+
+extract_mainnet_guestos_image(
```

Comment:
Applies to all: Do we want to mark these as manual? I think as repos, these wouldn't build unless pulled in. I think we want that behavior either way.